src/sys/resource.rs

```diff
@@ -0,0 +1,49 @@
+use std::mem;
+
+use libc::{self, c_int};
+pub use libc::{rlimit, RLIM_INFINITY};
+
+use {Errno, Result};
+
+#[repr(i32)]
```

Comment:
Also I kind of answered this in https://github.com/nix-rust/nix/pull/362#discussion_r61669895